### ISSUE TITLE
Missing Talkback description for viewing profile photo

### ISSUE THREAD

**Issue by clado** [signalapp/Signal-Android#12549]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
The button to view an other user's profile photo full screen is unlabelled in Talkback.

### Steps to reproduce
- view profile in conversation settings
- scroll over profile photo

**Actual result:** talkback says that button is unlabeled
**Expected result:** talkback should say that clicking on the profile photo will bring up a media preview

### Screenshots

![a screenshot of the user profile in the conversation settings view, with the user information censored. the user's photo has green talkback focus](https://user-images.githubusercontent.com/2933589/197419498-fc4d36aa-d7b0-4a49-8503-70751bf68752.png)

### Device info
**Device:** Google Pixel 2
**Android version:** Android 12 (LineageOS 19)
**Signal version:** 5.53.5

### Link to debug log
don't want to do this 😛 <3
